staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go

```diff
@@ -71,6 +71,13 @@ rules:
   - level: Metadata
 `
 
+var testCases = []struct {
```

Comment:
Since this is a set of testCases for a specific function, it's better to move it inside this function

Sorry for not mentioning this earlier